maven-plugin/src/main/java/org/moditect/mavenplugin/image/CreateRuntimeImageMojo.java

```diff
@@ -35,13 +35,13 @@
 @Mojo(name = "create-runtime-image", defaultPhase = LifecyclePhase.PACKAGE)
 public class CreateRuntimeImageMojo extends AbstractMojo {
 
-    @Parameter
+    @Parameter(defaultValue = "[]")
```

Comment:
This is to prevent another exception when no module paths are set. It's not really required since it internally adds the $JAVA_HOME module path to the user provided list.